package.json

```diff
@@ -4,6 +4,7 @@
   "engines": {
     "node": ">=20.1.0 <21"
   },
+  "packageManager": "yarn@1.22.22+sha256.c17d3797fb9a9115bf375e31bfd30058cac6bc9c3b8807a3d8cb2094794b51ca",
```

Comment:
still not clear about the hash, if a user has a malicious yarn version installed on their system it's already to late?

Reply:
My understanding is that `Corepack` ensures the right version is always used, and does the checksum check. You might not even need to have e.g. `yarn` installed locally.

Reply:
ah, interesting, so it's like a yarn lock. Makes sense, let's keep